src/schema.tsx

```diff
@@ -66,6 +66,7 @@ export const typeDefs = gql`
     url: String!
     label: String
     cmsId: ID
+    id: ID
```

Comment:
If we end up keeping `id` here instead of `cmsId` (see above) then I think it might be good to name this type explicitly to indicate it is adding a new _CMS_ Bookmark (as opposed to a custom one).